src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -98,10 +99,37 @@
      */
     private HashSet<String> navigatorProjects = new HashSet<>();
 
+    /**
+     * To store if project owner is group
+     */
+    private boolean isGroup;
+
+    /**
+     * To store the full name of the owner
+     */
+    private String ownerName = "";
+
+    /**
+     * To store if navigator should include subgroup projects
+     */
+    private boolean wantSubGroupProjects;
```

Comment:
This seems more like the kind of thing you'd want as a Trait

Reply:
Yes, but when I get the gitlabNavigatorContext with `GitLabSCMNavigatorContext navigatorContext = new GitLabSCMNavigatorContext().withTraits(navigator.getTraits());`, wantSubProjects is always `true`.